scripts/python_compliance.sh

```diff
@@ -16,50 +16,7 @@ else
     exit 1
 fi
 
-
-echo "Running auto-formatters"
```

Comment:
We should document somewhere how to run autoformatting now (and potentially, run it when running this script as before, unless you think something speaks against it)

Reply:
autoformatting is done automatically through the pre-commit hooks. You should stage your changes before running a git commit to be able to undo things. If you don't want the pre-commit hooks to run on every commit, you should alias your git commit command with a `-n` flag. That's nothing that should be configured centrally.

Reply:
I didn't want to include pylint in the pre-commit hooks, as it's prohibitively slow (20-30 sec.). The other hooks run in <5s total. It's still in the python_compliance script though (for CI)